tests/fixtures/bad_cases.expected.pest

```diff
@@ -7,4 +7,29 @@ arguments = {
 }
 blah      = {
     ("w") ~ &("z")
+}
+
+// https://github.com/pest-parser/pest-fmt/issues/15
+log_line1 = {
+    "0" ~ ("1" // a comment
+  | "2" // another comment
+  | "3" // longer comment
+  | "4" // final comment
+  | "a" | "b" | "c" | "d")
+}
+
+log_line2 = {
+    "0" ~ ("0" // a comment
+  | "0" // another comment
+  | "1" // longer comment
+  | "2" // final comment
+  | "a" | "b" | "c" | "d" | "e")
+}
+
+log_line3 = {
+  // lead comment
+    "1" ~ ("2" // another comment
+  | "3" // longer comment
+  | "4" // final comment
+  | "5" | "a" | "b" | "c" | "d" | "e") // comment
```

Comment:
will this split up the choices if there was more of them, so that it is not too long vertically?
The original feedback mentioned there may be too many choices:
>  the line that starts with pair35 in the output doesn't scale if there are lots and lots of rules.
In my real output - I ended up with a vertical formatted input with one of the output lines being 634 characters long (593 up to the comment).